WebARonARKit/WebARonARKit/ViewController.m

```diff
@@ -475,7 +475,7 @@ - (void)restartSession
 {
     ARWorldTrackingConfiguration *configuration =
         [ARWorldTrackingConfiguration new];
-    configuration.planeDetection = ARPlaneDetectionNone;// ARPlaneDetectionHorizontal;
+    configuration.planeDetection = ARPlaneDetectionHorizontal;
```

Comment:
What is this change for? Does it actually change functionality?

Reply:
It was a mistake. Fixed it.